packages/esm-patient-medications-app/src/order-basket/order-basket-search/order-basket-search-results.component.tsx

```diff
@@ -0,0 +1,177 @@
+    isLoading: isLoadingTemplates,
+    error: fetchingDrugOrderTemplatesError,
+  } = useDrugTemplate(drug?.uuid);
+  const { t } = useTranslation();
+  const config = useConfig() as ConfigObject;
+  const orderItems: Array<OrderBasketItem> = useMemo(
+    () =>
+      templates?.length
+        ? templates.map((template) => getTemplateOrderBasketItem(drug, config?.daysDurationUnit, template))
+        : [getTemplateOrderBasketItem(drug, config?.daysDurationUnit)],
+    [templates, drug, config?.daysDurationUnit],
+  );
+
+  const handleSearchResultClicked = (searchResult: OrderBasketItem, directlyAddToBasket: boolean) => {
+    onSearchResultClicked(searchResult, directlyAddToBasket);
+  };
+
+  return (
+    <>
+      {orderItems.map((orderItem, indx) => (
+        <ClickableTile
+          key={templates?.length ? templates[indx]?.uuid : drug?.uuid}
+          role="listitem"
+          className={isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`}
+          onClick={() => handleSearchResultClicked(orderItem, false)}
+        >
+          <div className={styles.searchResultTile}>
+            <div className={styles.searchResultTileContent}>
+              <p>
+                <strong>{drug?.display}</strong> {drug?.strength && <>&mdash; {drug?.strength}</>}{' '}
+                {drug?.dosageForm?.display && <>&mdash; {drug?.dosageForm?.display}</>}
+              </p>
+              {fetchingDrugOrderTemplatesError ? (
+                <p>
+                  <span className={styles.errorLabel}>
+                    {t('errorFetchingDrugOrderTemplates', 'Error fetching drug order templates')}
+                  </span>
+                </p>
+              ) : (
+                <p>
+                  {orderItem?.frequency?.value && <span className={styles.label01}>{orderItem?.frequency?.value}</span>}
+                  {orderItem?.route?.value && <span className={styles.label01}>&mdash; {orderItem?.route?.value}</span>}
+                </p>
+              )}
+            </div>
+            <Button
+              className={styles.addToBasketButton}
+              kind="ghost"
+              hasIconOnly={true}
+              renderIcon={(props) => <ShoppingCart size={16} {...props} />}
+              iconDescription={t('directlyAddToBasket', 'Immediately add to basket')}
+              onClick={() => handleSearchResultClicked(orderItem, true)}
+            />
+          </div>
+        </ClickableTile>
+      ))}
+    </>
+  );
+};
+
+const Skeleton = () => {
+  const isTablet = useLayoutType() === 'tablet';
+  return (
+    <div className={styles.searchResultSkeletonWrapper}>
+      <div className={styles.orderBasketSearchResultsHeader}>
+        <SkeletonText className={styles.searchResultCntSkeleton} />
+        <ButtonSkeleton size={isTablet ? 'md' : 'sm'} />
+      </div>
+      <Tile
+        className={`${isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`} ${
```

Comment:
This

```js
`${isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`} ${styles.skeletonTile}`
```

is rather reptitive - why not introduce a variable

```js
const cls = `${isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`} ${styles.skeletonTile}`;
```

and use this in `className={cls}`?